samples/04_gis_analysts_data_scientists/chennai_floods_analysis.ipynb

```diff
@@ -790,7 +790,7 @@
    "source": [
```

Comment:
<p>First, search for qualified satellite imagery (tiles) intersecting with the area of interest at the year 1991.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1126/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>